packages/@aws-cdk-testing/framework-integ/test/aws-codebuild/test/integ.docker-registry-build-images.ts

```diff
@@ -0,0 +1,162 @@
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import * as assert from 'assert';
+import * as cdk from 'aws-cdk-lib';
+import * as codebuild from 'aws-cdk-lib/aws-codebuild';
+import * as secretsmanager from 'aws-cdk-lib/aws-secretsmanager';
+import { Construct } from 'constructs';
+
+const app = new cdk.App();
+
+/*
+ * Deployment nstructions:
+ * 1. (Register and) log in to your Docker Hub account,
+ *  and generate a "Public Repo Read-only" Personal Access Token
+ *  - https://hub.docker.com/settings/security?generateToken=true
+ * 2. Update the `DOCKER_HUB_CREDENTIQLS` with your Docker Hub username and token
+ *  - DO NOT USE YOUR ACCOUNT PASSWORD, USE THE GENERATED TOKEN
+ * 3. Run the integration test
+ * 4. After the test is complete, invalidate the Personal Access Token you've just created
+ *  - https://hub.docker.com/settings/security?generateToken=true
+ */
+
+// https://hub.docker.com/settings/security?generateToken=true
+// Personal "Public Repo Read-only" token from Docker Hub.
+// The token was invalidated after snapshot creation
+const DOCKER_HUB_CREDENTIALS = JSON.stringify({
+  username: 'nmussy',
+
+  // !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+  // DO NOT USE YOUR ACCOUNT PASSWORD, USE A GENERATED PERSONAL ACCESS TOKEN
+  // !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+  password: // PASSWORD SHOULD NOT BE YOUR ACTUAL PASSWORD
+    'dckr_pat_I4j7TDEb_X92cW0MGB3FVZ_AlYc',
```

Comment:
This access token was mine, and has already been invalidated. I've already "unblocked" the secret on my personal repo, but it might be an issue again when we try to merge this?